vllm/model_executor/layers/quantization/compressed_tensors/compressed_tensors.py

```diff
@@ -118,6 +118,40 @@ def _is_dynamic_token_w8a8(self, weight_quant: BaseModel,
 
         return is_8_bits and is_token and is_symmetric and is_dynamic
 
+    def _is_fp8_w8a8(self, weight_quant: BaseModel,
+                     input_quant: BaseModel) -> bool:
+        # Confirm weights and activations quantized.
+        if weight_quant is None or input_quant is None:
+            return False
+
+        # Confirm we have floating points.
+        if not (weight_quant.type == QuantizationType.FLOAT
+                and input_quant.type == QuantizationType.FLOAT):
+            return False
+
```

Comment:
Could we clean this up a bit so it's easier to follow what is and what isn't supported?

Maybe starting with something like this for the lines above?
```
is_weight_quant_float = weight_quant and weight_quant.type == `QuantizationType.FLOAT`
is_input_quant_float = input_quant and input_quant.type == `QuantizationType.FLOAT`
```

Reply:
yep